front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -1,10 +1,6 @@
 import React, { memo, useCallback, MouseEvent } from 'react';
 
```

Comment:
Used the new InputFilterCollapsible component in the existing Topics filter just to make the UI cohesive for when we release this PR.

Any additional cleanup of existing code will happen when I work on the specific ticket :)